DashAI/back/plugins/utils.py

```diff
@@ -10,35 +14,30 @@ def _get_all_plugins() -> List[str]:
     List[str]
         A list with the names of all PyPI packages
     """
-    return [
-        "dashai-tabular-classification-package",
-        "pytorch",
-        "sklearn",
-    ]
+
+    client = xmlrpc.client.ServerProxy("https://pypi.python.org/pypi")
+    # get a list of package names
+    packages = client.list_packages()
+
+    return packages
 
 
 def _get_plugin_data(plugin_name: str) -> dict:
```

Comment:
Let's try to use the same names we have been using so far!

```suggestion
def _get_plugin_by_name_from_pypi(plugin_name: str) -> dict:
    # Also add a docstring!
```